src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/matcher/Matcher.kt

```diff
@@ -6,7 +6,7 @@ abstract class Matcher<T>(val items: List<T>?, protected val person: Person) {
   abstract fun isMatchingItem(item: T): Boolean
   abstract fun isPartialMatchItem(item: T): Boolean
 
-  fun isExactMatch() = !items.isNullOrEmpty() && items.size == 1 && items.any(::isMatchingItem)
+  open fun isExactMatch() = !items.isNullOrEmpty() && items.size == 1 && items.any(::isMatchingItem)
```

Comment:
premature generalisation? We do not take updates from HMCTS so we want to create a record where we do not already have one and ignore everything else.